spark/src/main/scala/geotrellis/spark/package.scala

```diff
@@ -70,11 +70,11 @@ package object spark
       new ContextRDD(rdd, metadata)
   }
 
-  type Component[T, C] = PLens[T, T, C, C]
+  type Component[T, C] = Lens[T, C]
 
   object Component {
     def apply[T, C](get: T => C, set: (T, C) => T): Component[T, C] =
-      PLens[T, T, C, C](get)(c => t => set(t, c))
+      Lens[T, C](get)(c => t => set(t, c))
```

Comment:
putting this change in another PR that will make RC2, can remove when you merge in master